demo-app/src/app/ot/playground/page.tsx

```diff
@@ -1,12 +1,50 @@
-import { parseYaml, applyConfiguration } from '@open-truss/open-truss'
+import {
+  parseYaml,
+  applyConfiguration,
+  type UqiMetadata,
+} from '@open-truss/open-truss'
 import * as COMPONENTS from '@open-truss-components'
+import createTrinoUqiClient from '@/lib/trino-uqi-client'
+
 const configurationFunction = applyConfiguration(COMPONENTS)
 
 async function PlaygroundPage(): Promise<JSX.Element> {
+  const trinoUqiClient = await createTrinoUqiClient({
+    auth: 'trino',
+    server: 'http://trino.orb.local:8080',
+  })
+  const queryIterator = await trinoUqiClient.query(
```

Comment:
Love this. Really slick / clean API and it's great that is going to be consistent across backend types